packages/renderer/src/lib/webview/Webview.svelte

```diff
@@ -37,6 +37,7 @@ let webviewElement: HTMLElement | undefined;
 const postMessageToWebview = (webviewEvent: unknown) => {
   const webviewEventTyped = webviewEvent as { id: string; message: unknown };
   if (id === webviewEventTyped.id) {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
probably use a valid type of webviewElement ? (or check if method send exists so we do not need to use any)